packages/plugins/EVM/src/state/TransactionFormatter/descriptors/ERC20.ts

```diff
@@ -25,6 +25,7 @@ export class ERC20Descriptor implements TransactionDescriptor {
                             title: 'Revoke',
                             description: 'Revoke the approval for the token.',
                             successfulDescription: 'Revoke the approval successfully.',
+                            failedDescription: 'Revoke requestion rejected.',
```

Comment:
Transaction rejected.

Reply:
The text on Figma design is 'Revoke requestion rejected.'